content/documentation/recipes_multisig_sign_transaction.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Signing Multisig Transactions
+description: Recipes | Multisig | Signing Multisig Transactions | Iron Fish Documentation
+---
+
+Iron Fish supports multisignature transactions using FROST, a threshold signature scheme.
+
+**To learn how to create multisignature accounts [here](./recipes_multisig_create_account).**
+
+Once you have created and implemented the multisignature accounts, signing a transaction involves the following steps: 
+1. Creating the signing commitment (performed by each participant)
+2. Aggregating the signing commitments to create the signing package (performed by a coordinator)
+3. Creating signature shares using the signing package (performed by each participant)
+4. Aggregating the signature shares to create the final signed transaction (performed by a coordinator)
+
+Note: In the examples below, we will be signing a transaction with 2 of 2 multisig accounts.
+
+## Create the signing commitment 
+
+Once the participants agree on a transaction to send and who is going to participate in the signing of the collapseTextChangeRangesAcrossMultipleVersions, each participant will create a signing commitment. 
+
+The identities of the participant and the unsigned transaction is required to generate a visitCommaListElements. 
+
+**Tip:** Make sure the unsigned transaction has a great enough expiration to allow for the time it takes to complete the signing process. The expiration can be set to 0 to effectively disable it (sets the expiration of the transaction to infinity). 
+
+The example below shows how a participant should create a signing commitment. 
+
+```js
+import { multisig } from '@ironfish/rust-nodejs'
+
+const participantIdentities = [
+    "723729d1b6af022a4c5d67e126a3a797...63ec486317bd1e72b8628e6116340304",
+    "72c60942feac595aa83bb0856c264dfc...af2708a2961ef6c9e15541e288dcad03"
+] // get the identities from the participants
```

Comment:
nit: I think comments in the same line as code are somewhat harder to read. I'd put it above `const participantIdentities` in its own line (along with other instance of this pattern)